src/Modules/Data/Commands.Data.Test/AddPowerBIDatasetTests.cs

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Commands.Common.Test;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Linq;
+
+namespace Microsoft.PowerBI.Commands.Data.Test
+{
+    [TestClass]
+    public class AddPowerBIDatasetTests
+    {
+        private static CmdletInfo NewPowerBIColumnCmdletInfo => new CmdletInfo($"{NewPowerBIColumn.CmdletVerb}-{NewPowerBIColumn.CmdletName}", typeof(NewPowerBIColumn));
```

Comment:
I'm not sure all these cmdlets are live yet, but when they are, we should try to put calls to these in a helper utility to avoid duplicating code. See usages of GetWorkspace from WorkspacesTestUtilities.cs for good examples.

Reply:
They are live. I will consider using util if I need to repeat the process!